mlrun/db/httpdb.py

```diff
@@ -3812,6 +3823,81 @@ def warn_on_s3_and_ecr_permissions_conflict(func):
                 default_docker_registry=config.httpdb.builder.docker_registry,
             )
 
+    def generate_event(self, name, event_data, project=""):
+        """
+        Generate an event.
+        :param name: name of the event.
```

Comment:
:param name: -- The name of the event.
        :param event_data: -- The data of the event.
        :param project: -- The project that the event belongs to.